test/system/700-play.bats

```diff
@@ -724,3 +724,12 @@ spec:
 
     run_podman kube down $pod_file
 }
+
+@test "podman kube with --authfile=/tmp/bogus" {
+    TESTDIR=$PODMAN_TMPDIR/testdir
+    mkdir -p $TESTDIR
+    echo "$testYaml" | sed "s|TESTDIR|${TESTDIR}|g" > $PODMAN_TMPDIR/test.yaml
+
+    run_podman 125 kube play --authfile=/tmp/bogus - < $PODMAN_TMPDIR/test.yaml
+    is "$output" "Error: stat /tmp/bogus: no such file or directory" "should faile with not such file"
```

Comment:
typo, "faile" (here and all of the above). Also, podman-remote usually has different error messages, so this might fail remote tests.